### PR TITLE
push observation submissions to kafka in all status

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: module/observationSubmissions/helper.js [hunk 1]
```diff
@@ -114,14 +114,25 @@ module.exports = class ObservationSubmissionsHelper {
...
                 }).lean();
 
+               
                 if (!observationSubmissionsDocument) {
-                    throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND+"or"+messageConstants.apiResponses.SUBMISSION_STATUS_NOT_COMPLETE;
+                    throw messageConstants.apiResponses.SUBMISSION_NOT_FOUND;
                 }
 
-                if( observationSubmissionsDocument.referenceFrom === messageConstants.common.PROJECT ) {
+                const solutionDocument = await database.models.solutions.findOne({
```
Comment (aks30) [inline module/observationSubmissions/helper.js:124]: @rakeshSgr - Use solution helper for the query

File: module/observationSubmissions/helper.js [hunk 2]
```diff
@@ -114,14 +114,25 @@ module.exports = class ObservationSubmissionsHelper {
...
+                    _id: observationSubmissionsDocument.solutionId,
+                }, { name: 1, scoringSystem : 1, description : 1, questionSequenceByEcm : 1 }).lean();
+                observationSubmissionsDocument['solutionInfo'] = solutionDocument;
+
+                const programDocument = await database.models.programs.findOne({
```
Comment (aks30) [inline module/observationSubmissions/helper.js:129]: @rakeshSgr - Use program helper for the query

File: module/observationSubmissions/helper.js (hunk 1 above)
Comment (rakeshSgr) [reply module/observationSubmissions/helper.js]: resolved

File: module/observationSubmissions/helper.js (hunk 2 above)
Comment (rakeshSgr) [reply module/observationSubmissions/helper.js]: resolved

**Pull request merged**